conanfile.py

```diff
@@ -320,6 +324,32 @@ def build_configure(self):
             if self.settings.os == "Windows":
                 args.append('--enable-libmfx' if self.options.qsv else '--disable-libmfx')
 
+            if tools.cross_building(self.settings):
+                args.append('--enable-cross-compile')
+                if self.settings.os == "Android":
+                    args.append('--target-os=android')
+                    if "android_ndk_installer" in self.deps_env_info:
+                        ndk_env_info = self.deps_env_info["android_ndk_installer"]
+                        cc = ndk_env_info.CC
+                        cxx = ndk_env_info.CXX
+                        ndk_root = ndk_env_info.NDK_ROOT
+                    else:
+                        env = os.environ
+                        cc = env["CC"]
+                        cxx = env["CXX"]
+                        ndk_root = env["NDK_ROOT"]
+                    args.append('--cc=%s' % cc)
+                    args.append('--cxx=%s' % cxx)
+                    arch = {'armv7': 'arm',
+                            'armv8': 'aarch64',
+                            'x86': 'i686',
+                            'x86_64': 'x86_64'}.get(str(self.settings.arch))
+                    abi = 'androideabi' if self.settings.arch == 'armv7' else 'android'
+                    cross_prefix = "%s/bin/%s-linux-%s-" % (ndk_root, arch, abi)
+                    args.append('--cross-prefix=%s' % cross_prefix)
+                    args.append('--arch=%s' % arch)
+                    args.append('--pkg-config=pkg-config')
```

Comment:
do we need to add `pkg-config_installer` to the build requirements? especially, for Windows